core/language/en-GB/Help/savewikifolder.tid

```diff
@@ -4,16 +4,27 @@ description: Saves a wiki to a new wiki folder
 <<.from-version "5.1.20">> Saves the current wiki as a wiki folder, including tiddlers, plugins and configuration:
 
 ```
---savewikifolder <wikifolderpath> [<filter>]
+--savewikifolder <wikifolderpath> [<filter>] [ [<name>=<value>] ]*
 ```
 
 * The target wiki folder must be empty or non-existent
 * The filter specifies which tiddlers should be included. It is optional, defaulting to `[all[tiddlers]]`
 * Plugins from the official plugin library are replaced with references to those plugins in the `tiddlywiki.info` file
 * Custom plugins are unpacked into their own folder
 
+The following options are supported:
+
+* ''filter'': a filter expression that defines the tiddlers to include in the output.
+* ''explodePlugins'': set to "no" to save plugins to the tiddlers directory of the target wiki folder, to suppress breaking plugins into their constituent shadow tiddlers (default is "yes").
```

Comment:
@Zacharia2 ... I did just translate this info to German, but it does not make sense to me. 

explodePlugins .. yes means ... convert prlugins to shadows ... BUT the text above says "no" will do so.  -> confused!

-------------

IMO the whole thins should be re-worded

```
* ''explodePlugins'': defaults to "yes"
** ''yes'' will "explode" plugins into shadow tiddlers and save them to the tiddlers directory of the target wiki.
** ''no'' will suppress breaking plugins into their constituent shadow tiddlers. 
```

That's how I think it should be written. That's how I think it works. Except it does not work that way. ..

Reply:
Hi @pmario

> explodePlugins .. yes means ... convert prlugins to shadows

the option relates to the savewikifolder command, so the wording refers to how plugins are handled by the savewikifolder command. The current behaviour is that tiddlers containing plugins are exploded into plugin folders containing individual tiddler files. The new option allows that behaviour to be suppressed, so that plugins are saved as JSON tiddler files.

Reply:
OK. Then it still should be written differently. eg:

```
* ''explodePlugins'': defaults to "yes"
** ''yes'' will "explode" plugins into shadow tiddlers and save them to a tiddlers plugin-directory
** ''no'' will suppress breaking plugins into their constituent shadow tiddlers. It will save the plugin as a single JSON tiddler
```

IMO that's much easier to read and understand. At least for me.

Reply:
Yes, this is clearer.